proto/private/dependencies.bzl

```diff
@@ -57,21 +120,68 @@ dependencies = {
             "https://github.com/bazelbuild/rules_python/archive/4b84ad270387a7c439ebdccfd530e2339601ef27.tar.gz",
         ],
     },
-    # Dependency of `com_google_protobuf`.
+    # Dependency of `com_github_protocolbuffers_protobuf`.
     "six": {
-        "build_file": "@com_google_protobuf//:third_party/six.BUILD",
+        "build_file": "@com_github_protocolbuffers_protobuf//:third_party/six.BUILD",
         "sha256": "d16a0141ec1a18405cd4ce8b4613101da75da0e9a7aec5bdd4fa804d0e0eba73",
         "urls": [
             "https://pypi.python.org/packages/source/s/six/six-1.12.0.tar.gz",
         ],
     },
-    # Dependency of `com_google_protobuf`.
+    # Dependency of `com_github_protocolbuffers_protobuf`.
     "zlib": {
-        "build_file": "@com_google_protobuf//:third_party/zlib.BUILD",
+        "build_file": "@com_github_protocolbuffers_protobuf//:third_party/zlib.BUILD",
         "sha256": "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1",
         "strip_prefix": "zlib-1.2.11",
         "urls": [
             "https://zlib.net/zlib-1.2.11.tar.gz",
         ],
     },
 }
+
+maven_dependencies = {
+    "com_google_protobuf_protobuf_java": {
+        "artifact": "com.google.protobuf:protobuf-java:3.10.0",
+        "sha1": "410b61dd0088aab4caa05739558d43df248958c9",
+        "sha1_src": "3ee94b1a2c1a74d9a27196e39c4bcf8dd0101e70",
+    },
+    "com_google_protobuf_protobuf_javalite": {
+        "artifact": "com.google.protobuf:protobuf-javalite:3.10.0",
+        "sha1": "a7e0350493910f45a208732992f8e519ade2fde5",
+        "sha1_src": "e43e8517029380185acc014f5dc6ee2ade484fb1",
+    },
+}
+
+def _protobuf_workspace_impl(ctx):
+    ctx.symlink(ctx.attr._build, "BUILD")
+
+protobuf_workspace = repository_rule(
+    implementation = _protobuf_workspace_impl,
+    attrs = {
+        "_build": attr.label(default = "@rules_proto//proto/private:BUILD.release"),
+    },
+)
+
+def _protoc_binary_impl(ctx):
+    args = ctx.actions.args()
+    args.add(ctx.file.src)
+    args.add(ctx.outputs.executable)
+    ctx.actions.run_shell(
+        inputs = [ctx.file.src],
+        outputs = [ctx.outputs.executable],
+        command = "cp $1 $2&&chmod +x $2",
```

Comment:
I'd rather not add a dependency on bash (because of Windows), but I can't think of a better way to copy binaries :/.

Reply:
I switched to a genrule with both cmd_shell and cmd_bat. Does that work? I don't have a way to test it on windows though.